examples/basics/plotting/grid_x_y_viewbox.py

```diff
@@ -0,0 +1,48 @@
+
+
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+"""
++---+-------------+
+|   |             |
+| y |   viewbox   |
+|   |             |
++---+-------------+
+| sp|     x       |
```

Comment:
Now this works, but I found a new bug: try making the window small enough that it's less than 50x50. We get some nasty error. That shouldn't happen. We might need to do a `try` clause if there isn't a cleaner way.